swift/Sources/Blockchains/Tezos.swift

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import Foundation
+
+public class Tezos: Blockchain {
+  /// Coin type for Level 2 of BIP44.
+  ///
```

Comment:
let's remove comments, they are specified in `Blockchain` class, the rest looks good